library/src/main/java/com/novoda/downloadmanager/DownloadMigratorBuilder.java

```diff
@@ -93,6 +93,9 @@ public DownloadMigrator build() {
         ServiceConnection serviceConnection = new ServiceConnection() {
             @Override
             public void onServiceConnected(ComponentName name, IBinder binder) {
+                if (!(binder instanceof LiteDownloadMigrationService.MigrationDownloadServiceBinder)) {
```

Comment:
Negative check on an early return? 🤔

Reply:
I thought the reason for using early returns was to avoid negative checks

Reply:
I can invert both if you would prefer?

Reply:
@pablisco yup though I tend to favour small if blocks though